source/fswatch.d

```diff
@@ -586,7 +614,44 @@ version (linux) unittest
 	ev = waitForEvent(watcher);
 	assert(ev.type == FileChangeEventType.remove);
 	assert(ev.path == "b.txt");
+
+	version (INotify)
+	{
+		// test for creation, modification, removal of subdirectory
+		mkdir("test2/subdir");
+		ev = waitForEvent(watcher);
+		assert(ev.type == FileChangeEventType.create);

```

Comment:
please test filepath aswell (should be `"test2/subdir"`) and the 2 after this `test2/subdir/c.txt`. And the one below that are commented but should test file paths like this too